src/structures/GuildMember.js

```diff
@@ -302,6 +302,17 @@ class GuildMember extends Base {
     return this.manageable && (this.guild.me?.permissions.has(Permissions.FLAGS.MODERATE_MEMBERS) ?? false);
   }
 
+  /**
+   * Whether this member is currently timed out
+   * @returns {boolean}
+   */
+  isTimedOut() {
+    return (
+      typeof this.communicationDisabledUntilTimestamp === 'number' &&
```

Comment:
You could probably remove this check since the next line will return false if communicationDisabledUntilTimestamp is null

Reply:
https://github.com/discordjs/discord.js/pull/7146#discussion_r775127198

Reply:
oops